web_src/js/features/issuesutil.js

```diff
@@ -0,0 +1,1471 @@
+  const simplemde = new SimpleMDE({
+    autoDownloadFontAwesome: false,
+    element: $editArea[0],
+    forceSync: true,
+    renderingConfig: {
+      singleLineBreaks: false
+    },
+    indentWithTabs: false,
+    tabSize: 4,
+    spellChecker: false,
+    toolbar: ['bold', 'italic', 'strikethrough', '|',
+      'heading-1', 'heading-2', 'heading-3', 'heading-bigger', 'heading-smaller', '|',
+      'code', 'quote', '|',
+      'unordered-list', 'ordered-list', '|',
+      'link', 'image', 'table', 'horizontal-rule', '|',
+      'clean-block', '|',
+      {
+        name: 'revert-to-textarea',
+        action(e) {
+          e.toTextArea();
+        },
+        className: 'fa fa-file',
+        title: 'Revert to simple textarea',
+      },
+    ]
+  });
+  $(simplemde.codemirror.getInputField()).addClass('js-quick-submit');
+  simplemde.codemirror.setOption('extraKeys', {
+    Enter: () => {
+      const tributeContainer = document.querySelector('.tribute-container');
+      if (!tributeContainer || tributeContainer.style.display === 'none') {
+        return CodeMirror.Pass;
+      }
+    },
+    Backspace: (cm) => {
+      if (cm.getInputField().trigger) {
+        cm.getInputField().trigger('input');
+      }
+      cm.execCommand('delCharBefore');
+    }
+  });
+  attachTribute(simplemde.codemirror.getInputField(), {mentions: true, emoji: true});
+  return simplemde;
+}
+
+function selectItem(select_id, input_id) {
+  const $menu = $(`${select_id} .menu`);
+  const $list = $(`.ui${select_id}.list`);
+  const hasUpdateAction = $menu.data('action') === 'update';
+
+  $menu.find('.item:not(.no-select)').on('click', function () {
+    $(this).parent().find('.item').each(function () {
+      $(this).removeClass('selected active');
+    });
+
+    $(this).addClass('selected active');
+    if (hasUpdateAction) {
+      updateIssuesMeta(
+        $menu.data('update-url'),
+        '',
+        $menu.data('issue-id'),
+        $(this).data('id'),
+        $(this).data('is-checked')
+      ).then(reload);
+    }
+    switch (input_id) {
+      case '#milestone_id':
+        $list.find('.selected').html(`<a class="item" href=${$(this).data('href')}>${
+          htmlEscape($(this).text())}</a>`);
+        // falls through
+      case '#project_id':
+        $list.find('.selected').html(`<a class="item" href=${$(this).data('href')}>${
+          htmlEscape($(this).text())}</a>`);
+        // falls through
+      case '#assignee_id':
+        $list.find('.selected').html(`<a class="item" href=${$(this).data('href')}>` +
+                      `<img class="ui avatar image" src=${$(this).data('avatar')}>${
+                        htmlEscape($(this).text())}</a>`);
+    }
+    $(`.ui${select_id}.list .no-select`).addClass('hide');
+    $(input_id).val($(this).data('id'));
+  });
+  $menu.find('.no-select.item').on('click', function () {
+    $(this).parent().find('.item:not(.no-select)').each(function () {
```

Comment:
```suggestion
    $(this).siblings('.item').each(function () {
```

Assuming this targets the siblings only

Reply:
Actually I see this is copied code, so can be disregarded.